<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>360</y>
      <width>300</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>310</width>
      <height>290</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>450</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Clicked</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_listen">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>320</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>listen</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuServer">
    <property name="title">
     <string>Server</string>
    </property>
   </widget>
   <addaction name="menuServer"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
